### ISSUE TITLE
Update readme to mention restart (or other action) required.

### ISSUE THREAD

**Issue by waynebloss** [coreybutler/nvm-windows#99]
I see some other issues that mention this in the comments, but no issue specifically for this request.

I ran the latest intaller (v1.1.0) on Windows 10 Pro (64-bit) and I observed that the symlinks and various environment variables were created. However, I could only run the `nvm` command in an administrator cmd prompt. Also, after installing any version of Node.js, no cmd prompt could locate the `node` or `npm` commands (admin or otherwise).

After a restart, I was able to run `nvm`, `node` and `npm` in all cmd prompts. I am still unsure if I should be using an administrative cmd prompt to run `nvm` at all. So I am just wondering if I can get some exact guidance on that. Also, I see that in #22, [dan-sterling says](https://github.com/coreybutler/nvm-windows/issues/22#issuecomment-65253093) to close Explorer. I am not sure if he means to end the task or just close any open Explorer windows.

In summary:
- Do I ever need to run `nvm` as admin?
- Should Explorer just be closed, killed or should the user logout or restart after install of nvm-windows?
- I think this info should be in the readme or elsewhere, did I miss something?

Thanks for your work!

**Comment by ts-web** [comment]
I encountered this problem while helping a colleague install nvm-windows. He had to restart for his command prompt to recognize the path update (?).

In order for CMD to recognize the `nvm` command, running `echo %PATH%` should output a string that contains `%NVM_HOME%;%NVM_SYMLINK%`. And echoing those variables should output the paths to the executables `nvm`, `node`, etc.

I found a StackOverflow question about (why CMD may not recognize path updates) [http://serverfault.com/questions/8855/how-do-you-add-a-windows-environment-variable-without-rebooting]
Maybe the MSI failed to broadcast a `WM_SETTINGCHANGE` event to explorer.

If anyone else has this problem, I'd like to see if the problem is solved by killing explorer.exe and restarting it. Or by running cmd from the system menu, not an explorer context menu. Those may be easier than a full restart. Also, is it enough to just log off and on again?

**Comment by JimBTek** [comment]
Restarting worked easily for me. sorry i did this before seeing ts-web's request

**Comment by ZebraFlesh** [comment]
Broadcasting the WM_SETTINGCHANGE method would be ideal here. Doing so would facilitate a silent installer. (I can't tell people or my build server to reboot in the middle of a build that happens to install this utility.)

**Comment by coreybutler** [comment]
This was added to the README, but somehow never closed.
